# Define the goal
string crop_waypoint  # Specify which dishwasher we want to use
---
# Define the result
float32 crop_estimated
---
# Define a feedback message
float32 percent_complete
